src/js/components/FilterButtons.js

```diff
@@ -62,19 +62,21 @@ class FilterButtons extends React.Component {
 }
 
 FilterButtons.defaultProps = {
-  renderButtonContent: function (title) {return title; },
-  onFilterChange: function () {}
+  inverseStyle: false,
+  onFilterChange: function () {},
+  renderButtonContent: function (title) {return title; }
```

Comment:
Please add an extra space before the `return` and make sure your editors linting is working properly. Thank you.